doc/release-notes-1.1.0.rst

```diff
@@ -0,0 +1,621 @@
+
+* :ref:`lib_spm`:
+
+  * Added a new non-secure-callable function :cpp:func:`spm_firmware_info`.
+
+* :ref:`lib_nrf_cloud`:
+
+  * Adopted to the new device shadow format.
+
+* at_host (``lib/at_host``):
+
+  * Updated to use a dedicated workqueue instead of the system workqueue.
+  * Miscellaneous fixes and improvements.
+
+* :ref:`at_cmd_parser_readme`:
+
+  * Refactored the library.
+  * Added :ref:`at_notif_readme` dispatcher.
+
+* LTE link control (``include/lte_lc.h``):
+
+  * Added PLMN lock option (default: false).
+  * Added PDN connection authentication option (default: false).
+  * Added modem modes without GPS (LTE-M or NB-IoT only).
+  * Added fallback to secondary LTE network mode (LTE-M/NB-IoT) if the device fails to connect using the primary network mode.
+  * Added a function to get the periodic TAU and active time settings from the current network registration status.
+  * Added a function to get the current functional mode.
+  * Added a function to get the current network registration status (not registered, roaming, registered home network).
+  * Added a function to get and set the system mode (LTE-M, NB-IoT, and GPS).
+  * Other minor improvements and fixes.
+
+
+Updated drivers
+---------------
+
+* :ref:`at_cmd_readme`: (? is that a driver or a library?)
+
+  * Added an option to initialize the driver manually.
+  * Fixed detection of CME / CMS errors.
+
+
+BSD library
+-----------
+
+* Updated the :ref:`nrfxlib:bsdlib` to version 0.4.3.
+* Updated bsdlib_init() to return the value of :cpp:func:`bsd_init` instead of (only) zero.
+* Added functionality that overrides untranslated errnos set by the BSD library with a magic word (0xBAADBAAD), instead of EINVAL, and prints a log message.
+  If ASSERTs are enabled, the application will assert.
+* Made DFU socket available through the socket offloading mechanism.
+* PDN socket option translation for in the socket offloading. (? what does this mean?)
+* Added offloading of local sockets.
+* Updated samples that use the BSD library to use ``CONFIG_NET_NATIVE=n`` to save RAM and ROM.
+
+Board support
+-------------
+
+* Thingy:91 (``nrf9160_pca20035``):
+
+  *  Removed support for earlier hardware versions of ``nrf52840_pca20035`` and ``nrf9160_pca20035``.
+     From |NCS| v1.1.0, only the latest hardware version is supported.
+  *  Removed configurations specific to the deprecated board versions from the :ref:`asset_tracker` application.
+
+
+Common libraries and drivers
+============================
+
+* Added the following libraries:
+
+  * :ref:`lib_dfu_target` - abstracts the specific implementation of how a DFU procedure is implemented.
+  * :ref:`fprotect_readme` - uses hardware (BPROT, ACL, or SPU) to protect flash areas from being changed.
+    This library is used by the immutable bootloader.
+
+* Added the following drivers:
+
+  * Flash patch driver - writes to UICR to disable flash patch functionality during the first boot of the image. (? where is this driver?)
```

Comment:
?

Reply:
nrf/drivers/flash_patch
nrf/include/drivers/flash_patch.h

Reply:
@sigvartmh could you comment on this?

Reply:
Not sure what you want me to comment. What oyvind said is true it is a driver  with no header. Which is loaded post_kernel init can be disabled through Kconfig.

Reply:
ah, sorry. Øyvind's comment didn't appear when I was reviewing :)